### DISCUSSION TITLE
Release plan for Q3 of 2021

### DISCUSSION THREAD

**Discussion by piotr-dziubecki** [casper-network/roadmap#23] [Announcements]
The release plan for 2021 Q3 has been published https://github.com/casper-network/roadmap/wiki/2021-Q4-Release-plan !

Your feedback is welcome. Please comment using this thread.

**Comment by KillianH** [comment]
Can you add more detail on the different issues to see what the features you refer to actually contain? 🙏

**Reply by piotr-dziubecki** [reply]
We will work on that, thank you.

**Reply by piotr-dziubecki** [reply]
@KillianH - we've updated some of the major epics. If you have more questions, let us know.

**Reply by KillianH** [reply]
Great I'll check it out :D

**Reply by KillianH** [reply]
I'm pretty curious about the Deploy acceptor  https://github.com/casper-network/roadmap/issues/20 and the light client https://github.com/casper-ecosystem/roadmap/issues/11 . Can do you have any details on those ? 

Also the link in the main post is wrong (Q4 instead of Q3 should be : https://github.com/casper-network/roadmap/wiki/2021-Q3-Release-plan ) 

Thank you for the updated content ! Love to see the fast sync feature :) This will help validators to recover quickly in case of a disaster.
